README.md

```diff
@@ -260,19 +260,60 @@ While using the repository from SBT, you can use any version of Apache Spark you
 * you run the sbt command using the wanted version `sbt -Dspark.version=1.4.0-SNAPSHOT`
 * there is a folder in `modules/spark/src/main/scala-2.1X` that points to the base of the required version (excl. the classifier SNAPHOST or RC): `spark-1.4`
 
+#### Building for specific distributions
+
+##### MapR
+
+##### Building for MapR
+MapR has a custom set of Hadoop jars that must be used.  To build using these jars:
+* Add the [MapR Maven repository](http://doc.mapr.com/display/MapR/Maven+Artifacts+for+MapR) as an sbt
+[proxy repository](http://www.scala-sbt.org/0.13/docs/Proxy-Repositories.html)
+* Build Spark Notebook using the MapR Hadoop jars by setting the `hadoop-version` property to the MapR-specific version.
+See the MapR Maven Repository link above for the specific versions to use.  For example, to build Spark Notebook with
+the MapR Hadoop jars and Hive and Parquet support for Spark 1.3.1:
+```
+sbt -Dspark.version=1.3.1 -Dhadoop.version=2.5.1-mapr-1503 -Dwith.hive=true -Dwith.parquet=true clean dist
+```
+
+##### Running with MapR
+A few extra jars need to be added to the app classpath:
+* `commons-configuration`
+* `hadoop-auth`
+* `maprfs`
+
+Then add them to the classpath in the `spark-notebook` script.  For example:
+```
+declare -r app_classpath="${HADOOP_CONF_DIR}:<long list of jars>:
```

Comment:
ah yes for the libs, I see.

However, you're right setting this var env with the add jars should work too. Might be worth trying so that there will be less manipulations :-s